tests/integration_cpp/basic.cpp

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) 2016-2020 Memgraph Ltd. [https://memgraph.com]
```

Comment:
This test should be named `basic_cpp.cpp` and be in the `integration` directory.
The existing test in the `integration` directory should be renamed to `basic_c.cpp`

Reply:
This isn't done.